UnifyArrayLengthsObject.js

```diff
@@ -0,0 +1,30 @@
+const _ = require("underscore")
+
+function unifyLengthArrays (obj) {
+  let maxLength = 0;
+
+  for (key in obj) {
+    if (obj[key].length > maxLength) {
+      maxLength = obj[key].length;
+    }
+  }
+
+  for (key in obj) {
+    if (obj[key].length < maxLength) {
+      const timesSmaller = Math.ceil(maxLength / obj[key].length);
```

Comment:
Hi @yjlim5,
Thank you for reviewing my code!
It is because I want to know how many times I can copy the elements of my smaller array until I reach the maxLength.
If for example my maxLength is 10 and if the length of my array in obj[key] is 2, I want to know how many times I can copy the array in obj[key] until I reach at least a length of 10. So 5 times.